scripts/update_tags_per_languages.py

```diff
@@ -0,0 +1,365 @@
+            else:
+                tags_fields[index] = f"{updated_tag}"
+        # updated tag is already in the field
+        # delete only instead of updating
+        else:
+            del tags_fields[index]
+    # old tag is not in the field
+    else:
+        # updated tag is not yet in the field
+        # add updated tag
+        if updated_tag not in tags_fields_lower:
+            # add space if the tag is not the first one in the string "do-not-add-space, add-space"
+            if tags_field_string != "":
+                tags_fields.append(f" {updated_tag}")
+            else:
+                tags_fields.append(f"{updated_tag}")
+        # final case, current tag missing and updated tag already in the field
+        # is equivalent to leave the field as is
+
+    tags_field_string = ",".join(tags_fields)
+    
+       
+    return tags_field_string
+
+
+def main():
+    for plural, singular in map_tags_field_url_parameter.items():
+        # 0) set variables
+        tags_list_url = f"https://{user}world.openfoodfacts.{env}/{plural}.json"
+        
+        # by default the query return 24 results. 
+        # increase to 1000 (so far chips in EN (should be crisps in EN)
+        #   had max number of products for categories with ~550)
+        products_list_for_tag_url = f"https://{user}world.openfoodfacts.{env}/{singular}/{{tag_id_placeholder}}.json?page_size=1000"
+        
+        file = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', f'taxonomies/{plural}.txt')) 
+        
+        # country is needed otherwise <plural>_lc will be "en"
+        post_call_url = f"https://{user}{{country}}.openfoodfacts.{env}/cgi/product_jqm2.pl"
+
+
+        # 1) get all tags
+        all_tags = get_from_api(tags_list_url)
+        # example:
+        #  all_tags = {
+        #      "tags": [
+        #          {"id": "it:frankreich", "known": 0, "name": "frankreich", ...}, 
+        #      ],
+        #  }
+
+        # 2) fetch unknown tags and look into taxonomy
+        possible_wrong_language_tags = unknown_tags_taxonomy_comparison(all_tags, file, plural)
+
+        # limit number of iterations
+        # for dev, number of elements in possible_wrong_language_tags 
+        #   can be changed in unknown_tags_taxonomy_comparison()
+        for current_tag, updated_tag in possible_wrong_language_tags.items():
+
+            # 3) get all products for this tag 
+            all_products_for_tag = get_from_api(products_list_for_tag_url.format(tag_id_placeholder=current_tag))
+            # example:
+            #  all_products_for_tag = {
+            #      "products": [
+            #          {"categories": "Lait", "categories_lc": "en", ...}, 
+            #      ],
+            #  }
+
+            if dev:
+                i = 0
+            for product in all_products_for_tag["products"]:
+                if dev:
+                    if i > 0:
+                        break
+                    i += 1
```

Comment:
It's ok to always count, and it's easier to read :-) 
```suggestion
            for i, product in enumerate(all_products_for_tag["products"]):
                if dev and i > 0:
                    break
```